src/resources/views/site/index.blade.php

```diff
@@ -0,0 +1,51 @@
+@extends('site.main')
+
+@section('body')
+    <div class="container">
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3>{{ trans('faq::faq.front.index-head') }}</h3>
+            </div>
+            <div class="panel-body">
+                @if(!$categories->isEmpty())
+                    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+                        @foreach($categories as $category)
+                            @if(!$category->questionsAnswers->isEmpty())
```

Comment:
I would put the part inside `foreach` into separate view. This view becomes too complicated.